### DISCUSSION TITLE
Unsupported stamp version or protocol

### DISCUSSION THREAD

**Discussion by bcookatpcsd** [DNSCrypt/dnscrypt-proxy#2071] [Q&A]
The website - https://dnscrypt.info/stamps/

Supports 'Plain DNS' as a protocol to generate stamps.

Making a stamp for an plain dns server and loading it into dnscrypt generates an error.. 

[2022-03-17 17:06:22] [FATAL] Stamp error for the static [10-20-8-29] definition: [Unsupported stamp version or protocol]

[static.10-20-8-29]
  stamp = 'sdns://AAAAAAAAAAAACjEwLjIwLjguMjk'

Does dnscrypt-proxy not support plain dns stamps?

**Comment by jedisct1** [comment]
`dnscrypt-proxy` intentionally doesn't support plain DNS.

DNS Stamps represent how to connect to a DNS relay or resolver. This is a generic specification, not specifically made for `dnscrypt-proxy`. It includes protocols that are not supported by this software.
